x-pack/test/functional/apps/ml/anomaly_detection/index.ts

```diff
@@ -11,17 +11,17 @@ export default function ({ loadTestFile }: FtrProviderContext) {
   describe('anomaly detection', function () {
     this.tags(['skipFirefox']);
 
-    loadTestFile(require.resolve('./single_metric_job'));
-    loadTestFile(require.resolve('./single_metric_job_without_datafeed_start'));
-    loadTestFile(require.resolve('./multi_metric_job'));
-    loadTestFile(require.resolve('./population_job'));
-    loadTestFile(require.resolve('./saved_search_job'));
-    loadTestFile(require.resolve('./advanced_job'));
-    loadTestFile(require.resolve('./single_metric_viewer'));
+    // loadTestFile(require.resolve('./single_metric_job'));
```

Comment:
Guess the plan is to fix the tests in a subsequent commit!?

Reply:
Fixed here [`ce159b0` (#97202)](https://github.com/elastic/kibana/pull/97202/commits/ce159b030e71c3cb69d3aaa79e48e5268b3b2eff)